src/Symfony/Component/Workflow/Marking.php

```diff
@@ -27,23 +27,58 @@ class Marking
     public function __construct(array $representation = [])
     {
         foreach ($representation as $place => $nbToken) {
-            $this->mark($place);
+            $this->mark($place, $nbToken);
         }
     }
 
-    public function mark(string $place): void
+    /**
+     * @param int $nbToken
```

Comment:
I suggest adding `@psalm-param int<1, max> $nbToken` (or `@psalm-param positive-int $nbToken` which is an alias for that range) to help static analyzers detect invalid usages of the argument (this is added in `@psalm-param` for now as we need `@param` to specify the future native type as long as we don't have a proper parameter for it yet).